scripts/install

```diff
@@ -102,7 +102,7 @@ ynh_setup_source --dest_dir="$final_path"
 if [ $path_url = "/" ]; then
   ynh_replace_string "__PATH_HACK__" "" "../conf/nginx.conf"
 else
-  ynh_replace_string "__PATH_HACK__" "$path_url" "../conf/nginx.conf"
+  ynh_replace_string "__PATH_HACK__" "$path_url/$path_url" "../conf/nginx.conf"
```

Comment:
Eh wokay that sounds weird but I suppose you know what you're doing ? :thinking:  (Isn't it a trouble that path_url starts with a slash so if it's equal to /toto it's gonna end up in `/toto//toto` (double slashes) ?

Reply:
Yes, that's the damn bug in Nginx... cf. the comment right above the "hack" thing: http://stackoverflow.com/a/35102259

It's that weird think, or a `rewrite` ; which is less efficient, I have read.